client/components/domains/connect-domain-step/connect-domain-step-login.jsx

```diff
@@ -43,7 +41,7 @@ export default function ConnectDomainStepLogin( {
 				{ sprintf(
 					/* translators: %s: the domain name that the user is connecting to WordPress.com (ex.: example.com) */
 					__(
-						'On your domain provider’s site go to the domains page. Find %s and go to it’s settings page.'
+						'On your domain provider’s site go to the domains page. Find %s and go to its settings page.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.